datajunction-server/datajunction_server/models/node.py

```diff
@@ -1127,6 +1127,9 @@ class DimensionAttributeOutput(SQLModel):
     """
 
     name: str
+    node_name: Optional[str]
```

Comment:
I think I agree with adding these attributes, but could you clarify your thinking for adding them here?

Reply:
So the main reason for adding them here is so that we can make a single call to retrieve common dimensions and get all the info we need to build this UI. Otherwise, we'll need to make multiple calls (as in, we can make a separate call for each dimension node and get their display names/PKs/etc), but that seems wasteful.

Reply:
Sounds good, so any changes on the dimension node should be reflected back here.